File: components_page/components/button/active_disabled.py
import dash_bootstrap_components as dbc
import dash_html_components as html

buttons = html.Div(
    [
        dbc.Button("Regular", color="primary", className="mr-1"),
        dbc.Button("Active", color="primary", active=True, className="mr-1"),
        dbc.Button("Disabled", color="primary", disabled=True),
    ]
)

File: components_page/components/button/active_disabled.jl
using DashBootstrapComponents, DashHtmlComponents

buttons = html_div([
    dbc_button("Regular", color = "primary", class_name = "me-1"),
    dbc_button("Active", color = "primary", active = true, class_name = "me-1"),
    dbc_button("Disabled", color = "primary", disabled = true),
]);
